### DISCUSSION TITLE
Why are Popper and Popover separate components? How are they different? When to use each?

### DISCUSSION THREAD

**Discussion by mstrater** [mui/base-ui#553] [Discussions]
From my understanding the main differences between them are that the Popover is modal component and the Popper supports modifiers (like the arrow nub) due to using Popper.js under the hood. Beyond this, they seem rather similar to me. Why not just have a Popper component with a "modal" property? When would you recommend using one over the other? In particular, since you provide a sample of how to achieve modal functionality with Popper using a ClickAwayListener, what advantages does Popover have over Popper?
Thanks!

**Comment by atomiks** [comment]
I know it's a bit confusing, but what you observe on the live Base UI site doesn't represent what will exist soon when we release the first alpha version. We're revamping the library entirely from its current version. They won't be separate in the new version.

**Reply by mstrater** [reply]
Exciting! Looking forward to the changes. :)

**Reply by mstrater** [reply]
@atomiks, I realized I mistakenly asked this under base-ui. Is it the same in mui-material?

**Reply by colmtuite** [reply]
The plan is to build Material UI v7—which will be released next year—on top of Base UI. So we will address any weirdness/confusion then.

**Reply by mstrater** [reply]
Thanks! Do you know yet what the new unified component will be called?

**Reply by colmtuite** [reply]
Popover
